services/command.py

```diff
@@ -24,11 +24,10 @@ async def handle(self, message: discord.Message):
             await message.reply(self.repository.response_for_command(trimmed_command))
             return
         self.logger.debug('Simple command not recognized')
-        if trimmed_command.startswith('dziekuje'):
-            self.logger.debug('Thank you command recognized')
-            self.helper.handle_thankyou(message)
-            return
+
+        #helper ranking print handling
         if trimmed_command == 'ranking':
+            #TODO add beautiful print
```

Comment:
takie komenty to też nie ma co zostawiać imo, na upiększanie printów jeszcze przyjdzie czas :D